buildenv/1.3.x/common/qt5/qt5.build

```diff
@@ -28,53 +29,32 @@ function prepare {
 
 	echo "# dummy" > qtbase/.gitignore
 
-	# Disable qmacpasteboardmime. There are symbol clashes with the 'cocoa' plugin,
-	# so seemingly, these two modules aren't currently engineered to be used in a
-	# static build together.
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-macextras-disable-qmacpasteboardmime.patch
-
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-force-qtimageformats-jasper-no-libjpeg.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-qtimageformats-jasper-4-color-rct-debian.patch
```

Comment:
I'm guessing you've just left out the three patches above because they aren't used on Windows... Or?...

Reply:
The first patch is not required anymore: https://bugreports.qt.io/browse/QTBUG-35310
The second and third patches should be updated.

Reply:
Are you sure it's fixed in MSVC 2017?

Reply:
Ah, per Thiago: "The fix is coming in 15.8." (from the linked bug)